js/main.js

```diff
@@ -0,0 +1,53 @@
+const NAMES_ARRAY = [
+  'Андрей','Мария',
+  'Станислав',
+  'Олег'
+];
+const COMMENTS_ARRAY = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё. Когда вы делаете фотографию, хорошо бы убирать палец из кадра.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.'
+];
+const DESCRIPTION = [
+  'Это мяч',
+  'Коты наше все',
+  'Анапа 2008',
+  '#Я в коридоре на Балли'
+];
+const OBJ = [];
+const getRandomPositiveInteger = (a, b) =>{
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+const createIdGenerator = () => {
+  let lastGeneratedId = 0;
+  return function (){
+    lastGeneratedId += 1;
+    return lastGeneratedId;
+  };
+};
+const randomIdNumber = createIdGenerator();
+const randomUrlNumber = createIdGenerator();
+const createObj = () => {
+  const randomDescriptionIndex = getRandomPositiveInteger(0, DESCRIPTION.length - 1);
```

Comment:
В целом все норм, можно оставить и так. Но, если есть желание, можно вынести в новую функцию универсальную по получаению случайного элемента массива, например такую:
```ts
const getRandomArrayElement = (array) => array[getRandomPositiveInteger(0, array.length - 1)];
```

и тогда будет чуть проще доставать случайные данные из массива
```suggestion
  const description = getRandomArrayElement(DESCRIPTIONS);
```

и при добавлении поля в результрующий объект сразу устанавливать это значение:
```ts
return {
    description: description,
    likes: randomCounterLikes,
    comments: COMMENTS_ARRAY[randomCommentsIndex],
    name: NAMES_ARRAY[randomNameIndex],
}
```


или вовсе не создавать изыбточную переменную и сразу записывать значения в поля объекта:
```ts
return {
    description: getRandomArrayElement(DESCRIPTIONS),
    likes: getRandomPositiveInteger(15,200),
   /* Тут оставльные поля*/
}
```

Можно оставить как и сейчас, если так понятнее, но, понять как это можно упростить -- желательно